src/cmd/build.rs

```diff
@@ -466,30 +471,33 @@ mod tests_ci_only {
                 ManifestPath::new(&path.join("new_project").join("Cargo.toml")).unwrap();
             let res = super::execute(
                 &manifest_path,
-                None,
+                Verbosity::Default,
```

Comment:
This is a required change, actually the code does not build without it. We just didn't notice because the `test-ci-only` feature was not included in the CI.